kura/distrib/src/main/resources/common/customize_iptables.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import sys
+import logging
+import os
+from network_tools import get_eth_wlan_interfaces_names
+
+def main():
+    logging.basicConfig(
+        format='[customize_iptables.py] %(asctime)s %(levelname)s %(message)s',
+        level=logging.INFO,
+        datefmt='%Y-%m-%d %H:%M:%S',
+        handlers=[
+            logging.StreamHandler()
+        ]
+    )
+    
+    iptables_filename = "/opt/eclipse/kura/.data/iptables"
```

Comment:
Use uppercase letters for constants.

Python doesn’t support constants or non-reassignable names. Using uppercase letters is just a convention, and it doesn’t prevent developers from assigning new values to your constant... but it's common enough to be a good idea to do it.

See https://peps.python.org/pep-0008/#constants

```suggestion
    IPTABLES_FILENAME = "/opt/eclipse/kura/.data/iptables"
```